packages/jest-core/package.json

```diff
@@ -13,6 +13,7 @@
     "ansi-escapes": "^4.2.1",
     "chalk": "^2.0.1",
     "exit": "^0.1.2",
+    "glob": "^7.1.3",
```

Comment:
we already depend on `micromatch` here, shouldn't need another glob module

Reply:
The difference is that glob doesn't just match strings but matches files in the FS - that's what I was referring to as extra FS calls

Reply:
I've reimplemented this with `micromatch`. Could you review please?